tests/i18n/tests.py

```diff
@@ -1918,10 +1918,18 @@ class NonDjangoLanguageTests(SimpleTestCase):
         LANGUAGES=[
             ('en-us', 'English'),
             ('xxx', 'Somelanguage'),
+            ('xyz', 'XYZ')
```

Comment:
Include a trailing comma so if more items are added later, we don't have to modify this line again.